concrete/config/app.php

```diff
@@ -1181,12 +1181,16 @@
 
     'api' => [
         'scopes' => [
```

Comment:
From what I can tell these are consumed and loaded into the database at install time. If that's the case we probably don't want them in config since changing their values has no effect.
Instead just use constants or a property or something.

I think this still needs more thought and deserves a service instead of just being arbitrary strings.